ckb-sync-mainnet/ansible/playbook.yml

```diff
@@ -33,6 +33,23 @@
       tags:
         - wait_ckb_synchronization
 
+    # run ckb replay from 1 to tip (based on sync db)
+    - name: Run and Wait for CKB replay
+      block:
+        - name: CKB Replay
+          shell: "CKB_LOG=error {{ ckb_workspace }}/ckb replay --tmp-target=/tmp --profile >> {{ ckb_data_dir }}/logs/run.log"
+        - name: Wait Until CKB replay from 1 to Tip
+          wait_for:
```

Comment:
`ckb replay` is a block-on command. So why use `wait_for` here?

Reply:
remove `wait_for`, done!